deploy-manage/distributed-architecture/shard-allocation-relocation-recovery/index-level-shard-allocation.md

```diff
@@ -3,27 +3,147 @@ mapped_urls:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules-allocation.html
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/shard-allocation-filtering.html
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/recovery-prioritization.html
+applies_to:
+  stack:
 ---
 
 # Index-level shard allocation
 
-% What needs to be done: Refine
+In Elasticsearch, per-index settings allow you to control the allocation of shards to nodes through index-level shard allocation settings. These settings enable you to specify preferences or constraints for where shards of a particular index should reside. This includes allocating shards to nodes with specific attributes or avoiding certain nodes. This level of control helps optimize resource utilization, balance load, and ensure data redundancy and availability according to your deployment's specific requirements. For additional details, check out:
 
-% Scope notes: also reference https://www.elastic.co/guide/en/elasticsearch/reference/current/allocation-total-shards.html and https://www.elastic.co/guide/en/elasticsearch/reference/current/data-tier-shard-filtering.html
+* [Shard allocation filtering](../../../deploy-manage/distributed-architecture/shard-allocation-relocation-recovery/index-level-shard-allocation.md): Controlling which shards are allocated to which nodes.
+* [Delayed allocation](../../../deploy-manage/distributed-architecture/shard-allocation-relocation-recovery/delaying-allocation-when-node-leaves.md): Delaying allocation of unassigned shards caused by a node leaving.
+* [Total shards per node](elasticsearch://reference/elasticsearch/index-settings/total-shards-per-node.md): A hard limit on the number of shards from the same index per node.
+* [Data tier allocation](asciidocalypse://docs/elasticsearch/docs/reference/elasticsearch/index-settings/data-tier-allocation-settings.md): Controls the allocation of indices to [data tiers](../../../manage-data/lifecycle/data-tiers.md).
```

Comment:
Looks like the right URL is https://github.com/elastic/elasticsearch/blob/main/docs/reference/elasticsearch/index-settings/data-tier-allocation.md
```suggestion
* [Data tier allocation](elasticsearch://reference/elasticsearch/index-settings/data-tier-allocation.md): Controls the allocation of indices to [data tiers](../../../manage-data/lifecycle/data-tiers.md).
```